### PR TITLE
Use builders linter if release has charts only

### PR DESCRIPTION
**Pull request by divolgin**
Implements a TODO comment.

If a release only has Helm charts, it should use Builders linter.

### CODE REVIEW

File: cli/cmd/release_lint.go [hunk 1]
```diff
@@ -127,10 +131,40 @@ func tarYAMLDir(yamlDir string) ([]byte, error) {
...
+		if err != nil {
+			return helmError
+		}
+
+		return nil
+	})
+
+	if err == helmError {
+		return false, nil
+	}
```
Comment (FourSigma) [inline cli/cmd/release_lint.go:163]: Do we want to return a `nil` error here?

Comment (divolgin) [reply cli/cmd/release_lint.go:163]: it's not an error.  it's the `false` result.

Comment (FourSigma) [reply cli/cmd/release_lint.go:163]:
```
_, err = loader.LoadFile(path)
```
We are basically using this call to see if this an Helm chart right? Is it possible returns other errors like "file not found"?  Additionally, do we need to wrap this helm error?

Comment (divolgin) [reply cli/cmd/release_lint.go:163]: If it can't be loaded then it's not a chart. There are no restrictions on what kinds of files can be passed in.

**Review by FourSigma — APPROVED** [review]

**Pull request merged**